Dockerfile

```diff
@@ -0,0 +1,83 @@
+# https://github.com/firebase/firebase-tools/issues/5614#issuecomment-1508515106
```

Comment:
The top half of this Dockerfile is to replicate the cloud function env.

Tried to cache the layers with https://github.com/docker/build-push-action but faced some issues with the registry permissions. For now it builds from scratch. Can add caching next time if we need it faster.